npm/iptm/ioshim.go

```diff
@@ -0,0 +1,13 @@
+package iptm
+
+import (
+	"io"
+	"os"
+)
+
+type IOShim interface {
+	GrabIptablesLocks() (*os.File, error)
```

Comment:
I am still thinking all of the methods in this interface can be isolated (i.e., non-capital). 
So, corresponding structs will be also isolated from other packages. 
Do I misunderstand something?